spec/features/address_bar_spec.rb

```diff
@@ -132,6 +132,48 @@ def query_string
         expect(page.current_url).not_to have_content("sb=0%2C0%2C10%2C10")
       end
     end
+
+    context 'setting granule filters' do
+      before(:all) do
+        visit '/search?q=C14758250-LPDAAC_ECS'
+        wait_for_xhr
+        dismiss_banner
```

Comment:
Don't use ```visit``` again here - this will remove the bounding box filter in the url. ```q``` will be the only filter left when the page loads.

Replace line 138 to 140 with this: (run it locally to make sure it works)
```
expect(page.current_url).to have_content("sb=0%2C0%2C10%2C10")
fill_in "keywords", with: 'C14758250-LPDAAC_ECS'
wait_for_xhr
```